importScripts/sql/cleaning/identifyImagesThatNeedReplacing.sql

```diff
@@ -0,0 +1,50 @@
+# Create a view to find object records that need replacing
```

Comment:
There are comments in the SQL which correspond to where I would normally have put PR comments.